packages/es-client/tests/test-granule-upsert.js

```diff
@@ -199,7 +199,6 @@ test('upsertGranule updates "failed" record for same execution', async (t) => {
   t.deepEqual(updatedEsRecord, {
     ...updates,
     _id: updatedEsRecord._id,
-    timestamp: updatedEsRecord.timestamp,
```

Comment:
See above. This seems sketchy but I think it's legit. We pass the timestamp in with the updated fields, not separately so I don't think it needs to be merged like this.